backend/ohq/statistics.py

```diff
@@ -232,3 +233,69 @@ def queue_calculate_questions_per_ta_heatmap(queue, weekday, hour):
         hour=hour,
         defaults={"value": statistic if statistic else 0},
     )
+
+
+def user_calculate_questions_asked(user):
+    num_questions = Decimal(Question.objects.filter(asked_by=user).count())
+    UserStatistic.objects.update_or_create(
+        user=user,
+        metric=UserStatistic.METRIC_TOTAL_QUESTIONS_ASKED,
+        defaults={"value": num_questions},
+    )
+
+
+def user_calculate_questions_answered(user):
+    num_questions = Decimal(Question.objects.filter(responded_to_by=user).count())
+    UserStatistic.objects.update_or_create(
+        user=user,
+        metric=UserStatistic.METRIC_TOTAL_QUESTIONS_ANSWERED,
+        defaults={"value": num_questions},
+    )
+
+
+
+def user_calculate_time_helped(user):
+    user_time_helped = (
+        Question.objects.filter(asked_by=user)
```

Comment:
Does this query work if the question was rejected or withdrawn? In that case the `time_response_started` field would be null. If it doesn't, perhaps filter for where the question status is answered?